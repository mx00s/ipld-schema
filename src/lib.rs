#![deny(clippy::all)]
#![deny(clippy::pedantic)]

pub mod schema;
