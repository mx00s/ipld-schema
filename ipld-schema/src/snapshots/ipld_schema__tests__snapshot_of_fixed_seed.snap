---
source: src/lib.rs
expression: "Seed::fixed()"
---
"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=".parse::<Seed>().unwrap()
